web/server/entrypoint.go

```diff
@@ -309,9 +307,8 @@ func (s *robotServer) serveWeb(ctx context.Context, cfg *config.Config) (err err
 				if errors.Is(err, context.Canceled) {
 					return
 				}
-				s.logger.Panicw("error creating restart checker", "error", err)
```

Comment:
This was super weird:
* We're panicing inside of a "PanicCapturingGo"
* "After" the panic we cancel a context.

Reply:
Yeah... good to get rid of these `Panic` logs; it's an odd function that does not seem particularly useful and is prone to weird "orphaned" logic like the context cancelation and return. I'm reminded of `log.Fatal` which notoriously calls `os.Exit(1)` and can surprisingly crash a whole program through the standard logging library.